api/src/main/java/io/strimzi/api/kafka/model/status/KafkaConnectS2Istatus.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.status;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+/**
+ * Represents a status of the Kafka Connect S2I resource
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "conditions", "observedGeneration", "httpRestApiAddress" })
+@EqualsAndHashCode
+public class KafkaConnectS2Istatus extends Status {
```

Comment:
* I think the S2I status could have some more useful information ... in particular the name of the build might be handy. 
* Also, maybe this can just extend the `KafkaConnectStatus` class?